Gruntfile.js

```diff
@@ -169,17 +169,37 @@ module.exports = function(grunt) {
         }
       }
     },
-      uglify: { 
-          options: { 
-              compress: true, 
-              mangle: true, 
-              sourceMap: false
-          }, 
-          target: { 
-              src: './dist/browser/js/ion-bundle.js',
-              dest: './dist/browser/js/ion-bundle.min.js',
-          }
+    uglify: {
+        options: {
+            compress: true,
+            mangle: true,
+            sourceMap: false
+        },
+        target: {
+            src: './dist/browser/js/ion-bundle.js',
+            dest: './dist/browser/js/ion-bundle.min.js',
+        }
+    },
+    mochaTest: {
```

Comment:
Configures the `grunt-mocha-test` package, which allows us to run Mocha tests via Grunt. Another file below (`mocha.opts`) configures the same settings, but Grunt can't use it directly.